.github/workflows/nightly_build.yml

```diff
@@ -79,6 +79,29 @@ jobs:
         export LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:${GITHUB_WORKSPACE}/bin/${KRATOS_BUILD_TYPE}/libs
         python3 kratos/python_scripts/testing/run_python_mpi_tests.py -l nightly -n 4 -f--oversubscribe
 
+    - name: Running MPI Python MultilevelMonteCarloApplication (2 Cores)
+      timeout-minutes : 10
+      run: |
+        . /opt/intel/mkl/bin/mklvars.sh
+        export PYTHONPATH=${PYTHONPATH}:${GITHUB_WORKSPACE}/bin/${KRATOS_BUILD_TYPE}
+        export LD_LIBRARY_PATH=/external_libraries/mmg/mmg_5_5_1/lib:/external_libraries/ParMmg_5ffc6ad/lib:${GITHUB_WORKSPACE}/bin/${KRATOS_BUILD_TYPE}/libs
+        mpiexec -np 2 python3 applications/MultilevelMonteCarloApplication/tests/test_MultilevelMonteCarloApplication_mpi.py -l nightly --using-mpi
+
+    - name: Running MPI Python MultilevelMonteCarloApplication (3 Cores)
+      timeout-minutes : 10
+      run: |
+        . /opt/intel/mkl/bin/mklvars.sh
+        export PYTHONPATH=${PYTHONPATH}:${GITHUB_WORKSPACE}/bin/${KRATOS_BUILD_TYPE}
+        export LD_LIBRARY_PATH=/external_libraries/mmg/mmg_5_5_1/lib:/external_libraries/ParMmg_5ffc6ad/lib:${GITHUB_WORKSPACE}/bin/${KRATOS_BUILD_TYPE}/libs
+        mpiexec --oversubscribe -np 3 python3 applications/MultilevelMonteCarloApplication/tests/test_MultilevelMonteCarloApplication_mpi.py -l nightly --using-mpi
+
+    - name: Running MPI Python MultilevelMonteCarloApplication (4 Cores)
+      timeout-minutes : 10
+      run: |
+        . /opt/intel/mkl/bin/mklvars.sh
+        export PYTHONPATH=${PYTHONPATH}:${GITHUB_WORKSPACE}/bin/${KRATOS_BUILD_TYPE}
+        export LD_LIBRARY_PATH=/external_libraries/mmg/mmg_5_5_1/lib:/external_libraries/ParMmg_5ffc6ad/lib:${GITHUB_WORKSPACE}/bin/${KRATOS_BUILD_TYPE}/libs
+        mpiexec --oversubscribe -np 4 python3 applications/MultilevelMonteCarloApplication/tests/test_MultilevelMonteCarloApplication_mpi.py -l nightly --using-mpi
```

Comment:
```suggestion
```
@philbucher shall I remove, correct? Please do not commit yet this change. I willdo once the old ParMmg `LD_LIBRARY_PATH` will be removed.

Reply:
yes now you can remove this since it is run automatically